QualityControl/public/pages/layoutListView/components/LayoutListCard.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
You are missing the CERN copy right documentation. Please see the other files as example